src/keys.rs

```diff
@@ -271,3 +274,19 @@ pub fn point_from_str(from: &str) -> anyhow::Result<Point> {
 
     Ok(point)
 }
+
+#[cfg(feature = "serde")]
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn ownership_public_key_deser_round() {
+        let pubkey = OwnershipKeyPair::new_random().public();
+
+        let str = serde_json::to_string(&pubkey).unwrap();
```

Comment:
`str` is not a keyword? Apparently not if it compiles 🙃

Reply:
GitHub format bug :)